templates/create_movies_form.html

```diff
@@ -8,15 +8,16 @@ <h1 class="mb-5">Create Movie Rating</h1>
 <!-- TODO: Feature 2 -->
 <div class="container">
     <form method="POST" action="/movies">
-
+        
+        <!-- input boxes for movie name, director, and rating -->
         <div class="form-group">
             <label for="movieName">Movie Name:</label>
-            <input type="text" class="form-control" id="movieName" name="movieName" aria-describedby="emailHelp" placeholder="Enter Movie Name" required>
```

Comment:
Way to fix update / fix the changes requested

Reply:
Thanks! Glad you were able to catch it.